fastlane_bot/helpers/tradeinstruction.py

```diff
@@ -163,6 +163,30 @@ def __post_init__(self):
         self._exchange_id = self.get_platform_id()
         self.custom_int = self.get_custom_int()
 
+    @property
+    def get_real_tkn_in(self):
+        """
+        Returns the correct token going into the trade.
+        """
+        if self.tknin_is_native:
+            return self.ConfigObj.NATIVE_GAS_TOKEN_ADDRESS
+        elif self.tknin_is_wrapped:
+            return self.ConfigObj.WRAPPED_GAS_TOKEN_ADDRESS
+        else:
+            return self.tknin
+
+    @property
+    def get_real_tkn_out(self):
+        """
+        Returns the correct token received by the trade.
+        """
+        if self.tknout_is_native:
+            return self.ConfigObj.NATIVE_GAS_TOKEN_ADDRESS
+        elif self.tknout_is_wrapped:
+            return self.ConfigObj.WRAPPED_GAS_TOKEN_ADDRESS
+        else:
+            return self.tknout
```

Comment:
Same code duplicated in two functions.
You can implement a single function which takes a token as input, then call it with `self.tknin` or `self.tknout` as needed.